### PR TITLE
support parameter collection for sqlserver

### PR DESCRIPTION
**Pull request by AngryMills**
### <support parameter collection for sqlserver>

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by wu-sheng** [comment]
You need to update test cases here, https://github.com/apache/skywalking/tree/master/test/plugin/scenarios/mssql-jdbc-scenario

**Comment by AngryMills** [comment]
> You need to update test cases here, https://github.com/apache/skywalking/tree/master/test/plugin/scenarios/mssql-jdbc-scenario
ok

File: apm-sniffer/apm-sdk-plugin/mssql-commons/src/main/java/org/apache/skywalking/apm/plugin/mssql/commons/PreparedStatementExecuteMethodsInterceptor.java [hunk 1]
```diff
@@ -47,6 +49,13 @@ public final void beforeMethod(EnhancedInstance objInst, Method method, Object[]
             Tags.DB_INSTANCE.set(span, connectInfo.getDatabaseName());
             Tags.DB_STATEMENT.set(span, SqlBodyUtil.limitSqlBodySize(cacheObject.getSql()));
             span.setComponent(connectInfo.getComponent());
+            if (JDBCPluginConfig.Plugin.JDBC.TRACE_SQL_PARAMETERS) {
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/mssql-commons/src/main/java/org/apache/skywalking/apm/plugin/mssql/commons/PreparedStatementExecuteMethodsInterceptor.java:52]: TRACE_SQL_PARAMETERS is false in default, you need to activate it in the test scenario. I can't see the change, could you recheck?

**Comment by AngryMills** [comment]
> You need to update test cases here, https://github.com/apache/skywalking/tree/master/test/plugin/scenarios/mssql-jdbc-scenario
I add a query method in scenario\src\main\java\org\apache\skywalking\apm\testcase\mssql\controller\CaseController.java and update the test\plugin\scenarios\mssql-jdbc-scenario\config\expectedData.yaml
Is this excepted?

File: apm-sniffer/apm-sdk-plugin/mssql-commons/src/main/java/org/apache/skywalking/apm/plugin/mssql/commons/PreparedStatementExecuteMethodsInterceptor.java (hunk 1 above)
Comment (AngryMills) [reply apm-sniffer/apm-sdk-plugin/mssql-commons/src/main/java/org/apache/skywalking/apm/plugin/mssql/commons/PreparedStatementExecuteMethodsInterceptor.java:52]: Maybe I can try. This is the test picture.

**Comment by wu-sheng** [comment]
Tests fail

**Comment by AngryMills** [comment]
> Tests fail
I will check it

**Comment by AngryMills** [comment]
> Tests fail
I check the CI log and I have no idea why nacos build failed
image

**Review by wu-sheng — APPROVED** [review]
Once tests passed, it is good for me.

**Pull request merged**